README.md

```diff
@@ -25,9 +26,22 @@ before publishing to operatorhub.io.
 
 Make the unreleased bundles available in Marketplace by adding the app registry:
 ```bash
+# Remove the hco-bundle from the community-operators sources
+$ kubectl get operatorsource -n openshift-marketplace community-operators -o yaml | sed "s/hco-operatorhub,//" | kubectl apply -f -
```

Comment:
again. why is a bundle not in community operators project being published here. Is this intentional?

Reply:
https://github.com/operator-framework/community-operators/tree/master/community-operators/hco-operatorhub

Ya this removes the hco-operator from the operator source, which will allow us to point to the unpublished bundle.  I'm not sure this will get a ton of use, but we'll need this for CI.